src/dotnet/commands/dotnet-msbuild/MSBuildForwardingApp.cs

```diff
@@ -31,7 +31,7 @@ public class MSBuildForwardingApp
             };
         
         private readonly IEnumerable<string> _msbuildRequiredParameters = 
-            new List<string> { "/m", "/v:m" };
+            new List<string> { "/m" };
```

Comment:
Indeed. Just `build` please, not `run`. We've gotten focused feedback since Preview 1 days that `run` was too chatty and **that** chattiness level was way below what a normal MSBuild output looks like.